robottelo/ui/rhai.py

```diff
@@ -27,23 +27,21 @@ def register_client_to_rhai(self, activation_key, org):
         self.vm.register_contenthost(activation_key, org)
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
-        # possible to sync the repo during the tests,
-        # adding a file in /etc/yum.repos.d/rhel6/7.repo
+        # possible to sync the repo during the tests, Adding repo file.
 
-        rhel6_repo = conf.properties['insights.rhel6_repo']
+        rhel6_repo = conf.properties['clients.rhel6_repo']
+        self.vm.configure_rhel_repo(rhel6_repo)
 
-        repo_file = (
-            '[rhel6-rpms]\n'
-            'name=RHEL6\n'
-            'baseurl={0}\n'
-            'enabled=1\n'
-            .format(rhel6_repo)
-        )
+        insights_repo = conf.properties['insights.insights_el6repo']
```

Comment:
The comment on the insights repo on the properties say that it is optional but this will make it required.